.github/workflows/deploy_docs_to_gh-pages.yml

```diff
@@ -0,0 +1,23 @@
+name: github pages
+
+on:
+  push:
+    branches:
+      - dev
+jobs:
+  deploy:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup mdBook
+        uses: peaceiris/actions-mdbook@v1
+        with:
+          mdbook-version: 'latest'
+
+      - run: mdbook build
+      - name: Deploy
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          publish_branch: gh-pages
+          publish_dir: ./book
```

Comment:
Should it be `./book` or `./docs`?

Reply:
./book

The ./docs dir has the markdown files and these gets bundled together to html files. So website is in ./book